aws/resource_aws_vpc_dhcp_options.go

```diff
@@ -182,7 +185,12 @@ func resourceAwsVpcDhcpOptionsRead(d *schema.ResourceData, meta interface{}) err
 
 func resourceAwsVpcDhcpOptionsUpdate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).ec2conn
-	return setTags(conn, d)
+
+	if err := setTags(conn, d); err != nil {
+		return err
+	}
+
+	return resourceAwsVpcDhcpOptionsRead(d, meta)
```

Comment:
Changed here to call `resourceAwsVpcDhcpOptionsRead()` as per standard pattern.

Reply:
Thanks!